docs/installing.rst

```diff
@@ -175,6 +175,7 @@ Now run cmake to create the makefiles. Replace the *installdir* argument to ``CM
 
 	cmake -G Ninja -DLLVM_ENABLE_ASSERTIONS=On '-DLLVM_ENABLE_PROJECTS=clang;lld'  \
 		-DLLVM_ENABLE_TERMINFO=Off -DCMAKE_BUILD_TYPE=Release \
+        -DLLVM_ENABLE_ZSTD=Off \
```

Comment:
Indentation seems wrong, also seems to confuse tabs vs. spaces here

Reply:
The diff seems wrong, but I swear I cannot see this in my computer.

![Screenshot 2023-02-24 at 12 36 41](https://user-images.githubusercontent.com/38472950/221220326-ae47a257-1594-4dde-b0ae-7db6ff42cb42.png)

Reply:
I replaced tabs by spaces, but nothing changed.

Reply:
That's super odd, in the github diff view there a two tabs but the line you introduced just has some spaces